### PR TITLE
task/DES-1964 - Publication Pipeline Wording Changes

### PR DESCRIPTION
**Pull request by fnets**
## Overview: ##
A lot of minor wording and formatting changes for the amend/versioning pipeline

## PR Status: ##

* [X] Ready.
* [ ] Work in Progress.
* [ ] Hold.

## Related Jira tickets: ##

* [DES-1964](https://jira.tacc.utexas.edu/browse/DES-1964)

## Summary of Changes: ##

- On the last step of the "publish" process for other-type projects, there is no warning message that shows "You cannot make changes after publishing."

- Removed the word "title" from the last bullet of the "Version" section of the area in the publication pipeline where a user chooses if they want to publish, amend, or version.

- Replaced Error, Success, and Note messages w/ icons

- Change the "Prepare to Publish" buttons to display "Prepare to Amend or Version" for other-type projects that already have DOIs

- Added clarifying text to the end of the Versioning pipeline

-  Formatted the File Selection step within the Versioning Pipeline correctly.

## Testing Steps: ##
1. Create a new other-type project and take it through the publication pipeline, and make sure that the applicable changes above are visible
2. DM Frank to be added to an other-type project with a dummy DOI and take it through the Amends pipeline,  and make sure that the applicable changes above are visible
3. Using the same project with the dummy DOI, go through the Versioning pipeline and make sure that the applicable changes above are visible.

### CODE REVIEW

**Review by jarosenb — CHANGES_REQUESTED** [review]
LGTM besides the requested change to `isOther` methods.

File: designsafe/static/scripts/data-depot/components/projects/pipeline-licenses/pipeline-licenses.component.html [hunk 1]
```diff
@@ -63,7 +63,7 @@
         <div style="padding-top: 40px; padding-bottom: 40px;">
             <p>
                 <h3 style="text-align: center">Select License(s)</h3>
-                <h3 style="text-align: center; color: orange;">
+                <h3 ng-if="!$ctrl.isOther()" style="text-align: center; color: orange;">
```
Comment (Strmiska) [inline designsafe/static/scripts/data-depot/components/projects/pipeline-licenses/pipeline-licenses.component.html:66]: I know it looks pretty ugly, but for the sake of making it easier to clean up later. Let's just make this expression `$ctrl.project.value.projectType === 'other'` That way when we're trying to find=>replace all these things we can get them in one swoop.

File: designsafe/static/scripts/data-depot/components/projects/pipeline-other/pipeline-other.component.html [hunk 2]
```diff
@@ -50,7 +50,7 @@
         <div style="padding-top: 40px; padding-bottom: 40px;">
             <p>
                 <h3 style="text-align: center">Proofread your Data</h3>
-                <h3 style="text-align: center; color: orange;">
+                <h3 ng-if="!$ctrl.isOther()" style="text-align: center; color: orange;">
```
Comment (Strmiska) [inline designsafe/static/scripts/data-depot/components/projects/pipeline-other/pipeline-other.component.html:53]: Same thing here

File: designsafe/static/scripts/data-depot/components/projects/pipeline-project/pipeline-project.component.html [hunk 3]
```diff
@@ -58,7 +58,7 @@
     <div style="padding-top: 40px; padding-bottom: 40px;">
         <p>
             <h3 style="text-align: center">Proofread your Project Metadata</h3>
-            <h3 style="text-align: center; color: orange;">
+            <h3 ng-if="!$ctrl.isOther()" style="text-align: center; color: orange;">
```
Comment (Strmiska) [inline designsafe/static/scripts/data-depot/components/projects/pipeline-project/pipeline-project.component.html:61]: Same thing here

**Review by Strmiska — CHANGES_REQUESTED** [review]
LGTM just swap out those if prjtype != other expressions and we're good!

**Review by Strmiska — APPROVED** [review]
LGTM!

**Review by jarosenb — APPROVED** [review]
LGTM!

**Pull request merged**